src/base/jsonwrapper.lua

```diff
@@ -0,0 +1,17 @@
+--
+-- jsonwrapper.lua
+-- Provides JSON encoding and decoding API by wrapping a third-party JSON library
+-- Copyright (c) 2017 Jason Perkins and the Premake project
+--
+
+	local implementation = dofile('json.lua')
+
+	json = {}
```

Comment:
I thought about that (and it actually was that way at first), but I felt like it would be better if it was global. The `http` API that we have is global, and if you're going to be making any kind of web API calls with that, you're probably going to need a JSON parser as well.